scss/atoms/typography/_mixins.scss

```diff
@@ -0,0 +1,26 @@
+// ==========================================================================
```

Comment:
> implementation instructions for mixins as well

These should probably eventually move to the documentation ( once such a thing exists ) but for now yes i agree.